src/sql/workbench/contrib/notebook/test/browser/htmlMarkdownConverter.test.ts

```diff
@@ -138,4 +138,12 @@ suite('HTML Markdown Converter', function (): void {
 		htmlString = '<a href="http://www.microsoft.com/images/msft.png">msft</a>';
 		assert.equal(htmlMarkdownConverter.convert(htmlString), '[msft](http://www.microsoft.com/images/msft.png)', 'Basic http link test failed');
 	});
+	test('Should transform <li> tags', () => {
```

Comment:
Can you add another test for ordered lists?

Reply:
> Does this work with numbered lists, too?

Yes it does.

![image](https://user-images.githubusercontent.com/40371649/98303987-f78f5d00-1f73-11eb-82a0-26e97a0d0078.png)

Reply:
Is that indented "1." expected behavior, or should we be switching to abc like in Word?

Reply:
I don't believe that's part of the markdown spec:

https://www.markdownguide.org/basic-syntax/#unordered-lists

Reply:
![image](https://user-images.githubusercontent.com/40371649/98308408-c8c9b480-1f7c-11eb-9bb5-0d2d4c30ec09.png)